ert3_examples/polynomial/experiments/test/experiment.yml

```diff
@@ -0,0 +1,2 @@
+type: sensitivity
```

Comment:
This could be worth keeping as an example if we rename it (could call it `partial_sensitivity` or similar), remove the incompatible records and make sure the `run_demo` script also runs this experiment...